### ISSUE TITLE
how to create pdf from an exist pdf file?

### ISSUE THREAD

**Issue by wzhsh90** [signintech/gopdf#170]
Hi:

is there a way to create pdf from an exist pdf file ？

**Comment by brunotrajano-mm** [comment]
It may be too late at this point, but there's an example within the package documentation <[https://github.com/signintech/gopdf#import-existing-pdf](https://github.com/signintech/gopdf#import-existing-pdf)> that could help.

**Comment by mr-bmt** [comment]
It may be too late at this point, but there's an example within the package documentation <[https://github.com/signintech/gopdf#import-existing-pdf](https://github.com/signintech/gopdf#import-existing-pdf)> that could help.

**Comment by denisbetsi** [comment]
Yes, though importing existing PDF has limitations. It'll strip out your all the links for example.

**Comment by vantaboard** [comment]
@oneplus1000 can you close this issue if it has been resolved?
